src/lib/components/nav/bar.svelte

```diff
@@ -240,7 +264,22 @@
 
       & :checked ~ .container {
         .pages {
+          width: 60%;
```

Comment:
can add transition to the backdrop? think it look better than immediate turn background black